<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>behave_qt_main_windowClass</class>
 <widget class="QMainWindow" name="behave_qt_main_windowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>790</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>behave_qt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="BehaveTree" name="behaveTree">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1920</width>
        <height>1200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="autoExpandDelay">
       <number>-1</number>
      </property>
      <property name="itemsExpandable">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>BehavePlus</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Surface Fire Spread Rate</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QStackedWidget" name="mainWindowStackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_1"/>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Fuel Model</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="fuelModelLineEdit">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>20</y>
          <width>150</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>1-h Moisture</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="oneHourMoistureLineEdit">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>50</y>
          <width>150</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>10-h Moisture</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>100-h Moisture</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="tenHourMoistureLineEdit">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>80</y>
          <width>150</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="hundredHourMoistureLineEdit">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>110</y>
          <width>150</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Live Herbaceous Moisture</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Live Woody Moisture</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="liveHerbaceousMoistureLineEdit">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>140</y>
          <width>150</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="liveWoodyLineEdit">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>170</y>
          <width>150</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Wind Speed</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="windSpeedLineEdit">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>210</y>
          <width>150</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Slope Steepness</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="slopeLineEdit">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>250</y>
          <width>150</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>381</width>
          <height>225</height>
         </rect>
        </property>
        <property name="title">
         <string>Fuel Input Method</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>181</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Fuel models (standard or custom)</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>200</y>
           <width>16</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>16</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>16</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>16</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>16</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>16</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>16</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>50</y>
           <width>211</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Fuel parameters (for custom fuel modeling)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>80</y>
           <width>291</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Two fuel models: 2D expected spread rate (recommended)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>110</y>
           <width>181</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Two fuel models: Harmonic mean</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>140</y>
           <width>241</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Two fuel models: Area weighted (like old BEHAVE)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>170</y>
           <width>331</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Special case fuel model: Palmetto-Gallberry (Hough and Albini, 1978)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>200</y>
           <width>341</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Special case fuel model: Western aspen (Brown and Simmerman 1986)</string>
         </property>
        </widget>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>300</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Fuel models (standard or custom)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fuel parameters (for custom fuel modeling)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Two fuel models: 2D expected spread rate (recommended)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Two fuel models: Harmonic mean</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Two fuel models: Area weighted (like old BEHAVE)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Special case fuel model: Palmetto-Gallberry (Hough and Albini, 1978)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Special case fuel model: Western aspen (Brown and Simmerman 1986)</string>
         </property>
        </item>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="spreadRateLineEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="calculateButton">
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BehaveTree</class>
   <extends>QTreeWidget</extends>
   <header>src/behave_qt/behave_qtree_widget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="behave_qt_main_window.qrc"/>
 </resources>
 <connections/>
</ui>
